content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -24,12 +24,21 @@ deprecated API versions to newer and more stable API versions.
 
 The **v1.26** release will stop serving the following deprecated API versions:
 
+#### Flow control resources {#flowcontrol-v126}
+
+The **flowcontrol.apiserver.k8s.io/v1beta1** API version of FlowSchema and PriorityLevelConfiguration will no longer be served in v1.26.
+
+* Migrate manifests and API clients to use the **flowcontrol.apiserver.k8s.io/v1beta2** API version, available since v1.23.
+* All existing persisted objects are accessible via the new API
+* No notable changes
+
 #### HorizontalPodAutoscaler {#horizontalpodautoscaler-v126}
 
 The **autoscaling/v2beta2** API version of HorizontalPodAutoscaler will no longer be served in v1.26.
 
 * Migrate manifests and API clients to use the **autoscaling/v2** API version, available since v1.23.
 * All existing persisted objects are accessible via the new API
+* No notable changes between **autoscaling/v2beta2** and **autoscaling/v2**
```

Comment:
This looks like a logically separate change.

Reply:
oops, sorry, was left in my working tree as a follow-up to https://github.com/kubernetes/website/pull/30776#discussion_r765157906

Reply:
dropped this line from this PR